global.json

```
@@ -1,5 +1,6 @@
 {
   "sdk": {
-    "version": "5.0.100"
+    "version": "5.0.100",
+    "rollForward": "latestMinor"
```

Revert unrelated change:

```suggestion
    "version": "5.0.100"
```

---

Seems to have been resolved via merge that is 3a40c13a0e86bb739847c70f19737b6fb5176d9a.